apps/dashboard/src/components/homepage/sections/PricingCard.tsx

```diff
@@ -69,13 +69,7 @@ export const PricingCard: React.FC<PricingCardProps> = ({
               {plan.title}
             </Heading>
             {current && (
-              <Badge
-                borderRadius="md"
-                size="label.sm"
-                px={3}
-                py={1.5}
-                textTransform="capitalize"
-              >
+              <Badge className="rounded-md px-3 py-1.5 capitalize">
                 Current plan
```

Comment:
Fix the vertical alignment 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/d88a4f90-9749-4269-a6ca-065fea9cacbe.png)